external/stb/stb_image_write.h

```diff
@@ -773,7 +773,9 @@ static int stbi_write_hdr_core(stbi__write_context *s, int x, int y, int comp, f
 #ifdef __STDC_LIB_EXT1__
       len = sprintf_s(buffer, sizeof(buffer), "EXPOSURE=          1.0000000000000\n\n-Y %d +X %d\n", y, x);
 #else
-      len = sprintf(buffer, "EXPOSURE=          1.0000000000000\n\n-Y %d +X %d\n", y, x);
+      len = snprintf(buffer, sizeof(buffer), "EXPOSURE=          1.0000000000000\n\n-Y %d +X %d\n", y, x);
```

Comment:
We've tried to keep from changing imported sources (VMA and STB image write) - can this be done some other way?

Reply:
I can
- Disable `-Wdeprecated-declarations`, as Apple blocks sprintf by flagging it deprecated
- Mark the stb include directory as a system include directory in cmake, which will disable warnings, but may also disable dependency calculations [according to cmake docs](https://cmake.org/cmake/help/latest/command/target_include_directories.html)

Preferences?

Reply:
I think maybe mark it as a system include directory and see what the result is there.  Thanks!

Reply:
Sounds good